tardis/tests/integration_tests/report.py

```diff
@@ -67,14 +70,16 @@ def __init__(self, dokuwiki_details):
         else:
             self.dokuwiki_url = dokuwiki_details["url"]
 
+    def pytest_sessionstart(self, session):
```

Comment:
Have you confirmed that the line has any effect?
Because I assume `super(DokuReport, self).pytest_sessionstart(session)` does the same thing.
This will essentially overwrite the previously defined `self.suite_start_datetime`